packages/types/src/base.ts

```diff
@@ -21,14 +21,19 @@ export enum ContractTypes {
 
 export enum ChainTypes {
   Ethereum = 'ethereum',
-  Bitcoin = 'bitcoin'
+  Bitcoin = 'bitcoin',
+  CosmosSDK = 'cosmosSdk'
```

Comment:
```suggestion
  Cosmos = 'cosmos'
```

Why did you choose to put `SDK` in the chain name?